config.h.cmake

```diff
@@ -462,4 +462,6 @@
 /* For default value of --early_plugin_load */
 #cmakedefine DEFAULT_EARLY_PLUGIN_LOAD @DEFAULT_EARLY_PLUGIN_LOAD@
 
+/* coredumper library */
+#cmakedefine HAVE_LIBCOREDUMPER 1
```

Comment:
This will define HAVE_LIBCOREDUMPER to 1 if HAVE_LIBCOREDUMPER is defined, regardless of its value. As it is always defined, it will always be defined.

A better approach is to use `cmakedefine01`, which defines it to 0 or 1 based on the cmake bool values. And then the later code should use `#if HAVE_LIBCOREDUMPER` instead of ifdef.

Reply:
Done